### PR TITLE
Added analytics for eksctl docs

### PR DESCRIPTION
**Pull request by Himangini**
### Description
I've setup analytics account and added the GA4 tag for tracking docs.

Closes https://github.com/weaveworks/eksctl-private/issues/420

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: userdocs/theme/partials/header.html [hunk 1]
```diff
@@ -75,5 +75,14 @@
       }
       toggleSwitch.addEventListener('click', switchTheme);
     </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-GBCDQ2QFTZ"></script>
```
Comment (nikimanoledaki) [inline userdocs/theme/partials/header.html:79]: Is it ok for the `GA_MEASUREMENT_ID` to be public? Do we want to pass it as an env var instead? I'm not sure 🤔

Comment (Himangini) [reply userdocs/theme/partials/header.html:79]: Nope that's completely fine

**Review by nikimanoledaki — APPROVED** [review]
LGTM (= Let's Get Those Metrics ha ha)! 🚀

File: userdocs/theme/partials/header.html [hunk 2]
```diff
@@ -75,5 +75,14 @@
...
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-GBCDQ2QFTZ');
```
Comment (cPu1) [inline userdocs/theme/partials/header.html:85]: Minor alignment nit:

```suggestion
        window.dataLayer = window.dataLayer || [];
        function gtag(){dataLayer.push(arguments);}
        gtag('js', new Date());
    
        gtag('config', 'G-GBCDQ2QFTZ');
```

**Review by cPu1 — APPROVED** [review]

**Pull request merged**